kebabpos/src/main/java/com/assemblypayments/kebabpos/Pos.java

```diff
@@ -698,12 +706,50 @@ private boolean processInput(String[] spInput) {
 
             case "rcpt_from_eftpos":
                 spi.getConfig().setPromptForCustomerCopyOnEftpos("true".equalsIgnoreCase(spInput[1]));
+                SystemHelper.clearConsole();
+                spi.ackFlowEndedAndBackToIdle();
+                printStatusAndActions();
+                System.out.print("> ");
                 break;
 
             case "sig_flow_from_eftpos":
                 spi.getConfig().setSignatureFlowOnEftpos("true".equalsIgnoreCase(spInput[1]));
+                SystemHelper.clearConsole();
+                spi.ackFlowEndedAndBackToIdle();
+                printStatusAndActions();
+                System.out.print("> ");
                 break;
 
+            case "print_merchant_copy":            	
+                spi.getConfig().setPrintMerchantCopy("true".equalsIgnoreCase(spInput[1]));
+                SystemHelper.clearConsole();
+                spi.ackFlowEndedAndBackToIdle();
+                printStatusAndActions();
+                System.out.print("> ");
+                break;
+               
+            case "receipt_header":
+            	String inputHeader = spInput[1].replace("\\r\\n", "\r\n");
+            	inputHeader = inputHeader.replace("\\\\", "\\");
+            	options.setCustomerReceiptHeader(inputHeader);
+            	options.setMerchantReceiptHeader(inputHeader);
+                SystemHelper.clearConsole();
```

Comment:
Please reformat this file with IntelliJ, as before.

Reply:
fixed